git-scp

```diff
@@ -91,7 +91,7 @@ maxfilelength=0
 filelength=0
 
 # extract added and modified files
-files_modified=$(echo "$changes" | sed -nE 's/^( )*[AM]+([ 	]+)//p' | tr -d "\"")
```

Comment:
@carsonreinke because added files has the prefix of '??' (double question mark) :)
![image](https://user-images.githubusercontent.com/415224/29651593-1bb539ce-88bf-11e7-8ca4-b597232970af.png)